src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonItem } from './PersonItem';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
you can set loading to true initially, then remove it `.finally`

Reply:
@choeqq isnt it whats already done?

Reply:
@choeqq i've changed it so that it initially is true and after resolving gets changed to false